docs/source/cdk/aws-setup.rst

```diff
@@ -0,0 +1,43 @@
+AWS Setup
+=========
+Download/requirements
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Ensure you have installed nodejs (newer than version 14), AWS CLI, and Docker
+
+- `nodejs <https://nodejs.org/en/download/>`_
+- `AWS CLI <https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html>`_
+- `Docker <https://docs.docker.com/get-docker/>`_
+
+New User
+~~~~~~~~~
+#. Log-in to `AWS console <https://aws.amazon.com/console/>`_
+#. Use *imap-sdc-development* for Account ID
+#. Enter user name and password
+#. Select IAM from *Services* menu or search IAM
+#. Select *Users* from left menu
+#. Click your user button
+#. Select *Security Credentials* tab
+#. Click *Create Access Key*
+#. Make note of *Access Key ID* and *Secret Access Key*
+#. You can download the .csv file if you want
+#. Click *Close*
+
+Existing User
+~~~~~~~~~~~~~
+#. In command line, run the following command to set up the aws environment::
+
+    aws configure
+
+#. Enter the *Access Key ID* and *Secret Access Key* from the previous step
```

Comment:
```suggestion
#. Enter your *Access Key ID* and *Secret Access Key* (If you don't have these, follow the :ref:`aws-new-user` section)
```
Adding a restructured text link to the previous section would be nice too. https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#ref-role

```python
# just above the previous section
.. _aws-new-user:
```

Reply:
I cant seem to get this. It might be a "my.pars" or something to do with the conf.py file